manager/manager.go

```diff
@@ -726,6 +726,9 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 	if strPassphrase == "" {
 		return nil
 	}
```

Comment:
I just realized this function does nothing if the current passphrase is empty. Is it impossible to switch back to an unencrypted key? Should we add support for that?